src/actionCreators/index.js

```diff
@@ -103,3 +106,18 @@ export const confirmEmail = (userId, email, jwt) => {
       });
   };
 };
+
+export const inviteUser = (userId, email, invitationCode) => {
+  return dispatch => {
+    dispatch(inviteUserStart());
+    axi
+      .post(API.INVITE_USER(userId), { email, code: invitationCode })
+      .then(res => {
+        return dispatch(inviteUserSuccess());
+      })
+      .catch(err => {
+        const errMsg = 'Fail to invite the user.';
```

Comment:
works for now but if this is a user error message maybe we could send them something less robot sounding